lessons/1_setup/index.html

```diff
@@ -145,43 +145,103 @@ <h3>What could be a Final Project?</h3>
           </ul>
         </section>
         <section>
-          <h3>Laptop checkup</h3>
+          <h3>Laptop checkup (software)</h3>
           <ul class="fragment">
-            <li>
-              Firefox:
+            <li class="fragment">
+              Firefox (option A):
               <a href="https://www.mozilla.org/firefox/new/"
                 >https://www.mozilla.org/firefox/new/</a
               >
-            </li> or
-            <li>
-              Chrome:
+            </li>
+
+            <li class="fragment">
+              Chrome (option B):
               <a href="https://www.google.com/chrome/"
                 >https://www.google.com/chrome/</a
               >
             </li>
-            <li>
-              VSCode*:
+            <li class="fragment">
+              <a href="https://git-scm.com/download/win"
+                >Git (only for windows)</a
+              >
+              <p><small>Mac and linux already installed by default</small></p>
+            </li>
+            <li class="fragment">
+              VSCode:
               <a href="https://code.visualstudio.com/download"
                 >https://code.visualstudio.com/download</a
-              ><small>* ask us about the recommended plugins</small>
+              ><small
+                >Recommended plugins
+                <ol>
+                  <li>bäumer.vscode-eslint</li>
+                  <li>esbenp.prettier-vscode</li>
+                  <li>bäumer.vscode-eslint</li>
+                  <li>ms-vsliveshare.vsliveshare</li>
+                  <li>ms-vsliveshare.vsliveshare-pack</li>
```

Comment:
PKief.material-icon-theme that one is good as well, adds nice icons

Reply:
oh that's nice i didn't know that one. Maybe we can use this as a chance to give a small tutorial to the students on how to set it up nicely :)

Reply:
yep, sure :)

Reply:
done